Source/Core/Common/SymbolDB.h

```diff
@@ -83,6 +85,9 @@ class SymbolDB
 	virtual Symbol *AddFunction(u32 startAddr) { return nullptr; }
 
 	void AddCompleteSymbol(const Symbol &symbol);
+	
+	void PopulateSymbolComments();
+	const std::string GetCommentFromAddr(u32 addr);
```

Comment:
This should just be `std::string`, otherwise you're referencing a destructed object in the event empty string is returned